testing/fil_token_integration/actors/basic_token_actor/src/lib.rs

```diff
@@ -46,23 +47,50 @@ impl FrcXXXToken<RuntimeError> for BasicToken<'_> {
         Ok(self.util.balance_of(&params)?)
     }
 
+    fn transfer(&mut self, params: TransferParams) -> Result<TransferReturn, RuntimeError> {
+        let spender = caller_address();
+        self.util.transfer(&spender, &spender, &params.to, &params.amount, &params.data)?;
+
+        // FIXME: populate with real values
```

Comment:
The library function here would need to return the new balances and remaining allowance in order to support this return type well